config/opensearch_dashboards.yml

```diff
@@ -29,6 +29,10 @@
 # dashboards. OpenSearch Dashboards creates a new index if the index doesn't already exist.
 #opensearchDashboards.index: ".opensearch_dashboards"
 
+# OpenSearch Dashboards uses an index in OpenSearch to store dynamic configurations.
+# This shall be a different index from opensearchDashboards.index.
+# opensearchDashboards.config_index: ".opensearch_dashboards_config"
```

Comment:
Since this index is to store app settings and used to do hot reload. Not sounds like configuration usage. 

To make it differentiate from a yml configuation,  maybe rename it to `opensearchDashboards.setting_index`?

Reply:
The difference between config and settings is config is for application level, settings is for user level

Reply:
Thanks @seraphjiang for the explanation!